### DISCUSSION TITLE
Contributor License Agreement

### DISCUSSION THREAD

**Discussion by jdsika** [foxglove/mcap#1203] [General]
Hi team,

MCAP is still referencing a CLA in its readme [here](https://github.com/foxglove/mcap?tab=readme-ov-file#license):

> License
> [MIT License](https://github.com/foxglove/mcap/blob/main/LICENSE). Contributors are required to accept the [Contributor License Agreement](https://github.com/foxglove/cla).

The CLA repository does not exist anymore. Was this requirement removed from the MCAP project as it was originally designed to protect the right of Foxglove to conduct a relicensing on Foxglove Studio and is now considered not needed anymore?

We at BMW would highly appreciate following a standard open source process within the MCAP repository which does only e.g. require a standard DCO in combination with the MIT license (and a code of codunct plus contributing guidelines). We are currently evaluating the legal requirements for a compliant collaborative work on MCAP. This would allow us to promote the usage of the MCAP format especially in the landscape of European research projects.

I would be happy to engage in a discussion regarding those requirements via email/teams and document the results here for the community to see. Feel free to contact me carlo.van-driesten@bmw.de

Best regards
Carlo

**Comment by amacneil** [comment]
Hi @jdsika 

We recently removed the CLA requirement from MCAP and other Foxglove repos. I didn't see that it was still referenced in the readme, but have updated it along with some additional context.

- https://github.com/foxglove/mcap/pull/1207

**Comment by jdsika** [comment]
Thank you very much! I think it will help the adoption of MCAP. I am looking forward to deepen the collaboration.

One side note:
When citing the project you could think about a formulation that clearly states the original authorship of Foxglove but expresses the transition into a collaborative effort.
